# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1326135184 28800
#      Mon Jan 09 10:53:04 2012 -0800
# Branch stable
# Node ID 3f2a3076b9a642319294bf5686754e49d0e6b861
# Parent  6b79540a0e5280d09963b9dc5476c0ad1d563645
NEWS: Update list of functions added for 3.6.0 release.

* NEWS: Update list of functions added for 3.6.0 release.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -47,17 +47,17 @@ Summary of important user-visible change
       +=  -=  .+=  .-=  .*=  ./=  .\=  .^=  .**=  &=  |=
 
     See the "Broadcasting" section in the new "Vectorization and Faster
     Code Execution" chapter of the manual for more details.
 
  ** Octave now features a profiler, thanks to the work of Daniel Kraft
     under the Google Summer of Code mentorship program.  The manual has
     been updated to reflect this addition.  The new user-visible
-    functions are profexplore, profile, and profshow.
+    functions are profile, profshow, and profexplore.
 
  ** Overhaul of statistical distribution functions
 
     Functions now return "single" outputs for inputs of class "single".
 
     75% reduction in memory usage through use of logical indexing.
 
     Random sample functions now use the same syntax as rand() and accept
@@ -166,20 +166,22 @@ Summary of important user-visible change
     functions nor the waitfor function.
 
  ** New keyword parfor (parallel for loop) is now recognized as a valid
     keyword.  Implementation, however, is still mapped to an ordinary
     for loop.
 
  ** Other new functions added in 3.6.0:
 
-      is_dq_string    nthargout    usejava     
-      is_sq_string    python       waitbar
-      narginchk       recycle      zscore            
-    
+      bicg                       nthargout                   usejava
+      is_dq_string               narginchk                   waitbar
+      is_sq_string               python                      zscore
+      is_function_handle         register_graphics_toolkit 
+      loaded_graphics_toolkits   recycle                   
+
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.2 and have been
     removed from Octave 3.6.
                                            
       create_set          spcholinv    splu   
       dmult               spcumprod    spmax
       iscommand           spcumsum     spmin
